perf/src/main/java/org/polyfill/perf/components/TablePrinter.java

```diff
@@ -0,0 +1,44 @@
+package org.polyfill.perf.components;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+
+/**
+ * Created by smo on 6/9/17.
+ */
+public class TablePrinter {
```

Comment:
This was used for printing results to tsv/csv. We can use this later if we decide to do that. If not, we can remove this.